conf/machine/p3768-0000-p3767-0003.conf

```diff
@@ -0,0 +1,24 @@
+#@TYPE: Machine
+#@NAME: Nvidia Jetson Orin Nano 8GB (P3767-0003)
+#@DESCRIPTION: Nvidia Jetson Orin Nano 8G module in P3768 carrier
+
+TEGRA_FAB ?= "300"
+TEGRA_BOARDSKU ?= "0003"
+TEGRA_BOARDREV ?= "P.1"
+
+TEGRA_BUPGEN_SPECS ?= "fab=300;boardsku=0003;boardrev=;chipsku=00:00:00:D5;bup_type=bl \
+                       fab=300;boardsku=0003;boardrev=;bup_type=kernel"
+
+TNSPEC_BOOTDEV ?= "nvme0n1p1"
+PARTITION_LAYOUT_TEMPLATE_DEFAULT ?= "flash_t234_qspi.xml"
+PARTITION_LAYOUT_TEMPLATE_DEFAULT_SUPPORTS_REDUNDANT ?= "1"
+PARTITION_LAYOUT_EXTERNAL_DEFAULT ?= "flash_l4t_t234_nvme.xml"
+TEGRAFLASH_NO_INTERNAL_STORAGE = "1"
+
+MACHINEOVERRIDES =. "orin-nano-som:"
```

Comment:
Having an extra override for the SoM is OK with me, and we should probably do the same for all of the SKUs, but could you name it `p3767-0003` instead of `orin-nano-som`, to disambiguate between this one and SKU 4 (the 4GB SoM)?